src/Evented.ts

```diff
@@ -36,6 +36,22 @@ function handlesArraytoHandle(handles: Handle[]): Handle {
 	};
 }
 
+export interface EventObject {
+	/**
+	 * The type of the event
+	 */
+	readonly type: string | symbol;
```

Comment:
I've updated the internal references to `EventObject` to use the local interface.